### ISSUE TITLE
new homepage a11y issues

### ISSUE THREAD

**Issue by aaronhans** [cagov/covid19#1401]
New homepage needs to:
- get 100 on Lighthouse accessibility audit
- get perfect score with raw axe audit (I thought lighthouse was based on this but more issues from axe are reported even after lighthouse 100 achieved)
- page is fully keyboard navigable

Outstanding issues:

- [x] element fails aria roles, make keyboard navigable or hidden: div.hero-alert
- [x] element fails aria roles, make keyboard navigable or hidden: section.news.trigger
- [x] element fails aria roles, make keyboard navigable or hidden: nav.footer-section-nav
- [x] nav not closeable with esc key

**Comment by kkoryaka** [comment]
Got 100% on Lighthouse accessibility audit.
Axe wants me to go to Is this page helpful component.

**Comment by aaronhans** [comment]
nice @koryfey!

**Comment by aaronhans** [comment]
Latest "Is this page userful" component HTML is here: https://github.com/cagov/Alpha/blob/master/components/pagerating/template.js is this what we need to change for axe @koryfey?

**Comment by kkoryaka** [comment]
Page is fully keyboard navigable except expanded nav is not closing on escape key. Need to take care of that.

**Comment by xjensen** [comment]
@kkoryaka I went ahead and added escape key behavior to the expanded nav menu.

**Comment by aaronhans** [comment]
Thanks for fixing these @koryfey and @xjensen
